src/components/WebGLView/WebGLView.tsx

```diff
@@ -342,6 +342,12 @@ export const WebGLView = connector(
 
     onMouseDown(event) {
       event.preventDefault();
+      
+      // call custom hook
+      if(this.props?.overrideComponents?.mouseInteractionHooks["mousedown"] != null){
+        const coords = CameraTransformations.screenToWorld(this.mouseController.currentMousePosition, this.createTransform());
+        this.props?.overrideComponents?.mouseInteractionHooks["mousedown"](coords)
```

Comment:
["mousedown"] syntax is wrong here since it has no static inference, always use .mousedown syntax